README.md

```diff
@@ -52,7 +52,7 @@ python3 -m venv .venv
 source .venv/bin/activate
 
 # Install Modular Adversarial Robustness Toolkit, if you plan to create your own `configs` folder elsewhere.
-pip install -e .
+pip install -e .[core,vision,developer]
```

Comment:
you can create a `full` dependency that basically installs everything.

Reply:
Fixed in 68425bc4aaddc35055ca6e5cd4353dab2f1caab5